src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java

```diff
@@ -156,7 +158,7 @@ private static boolean isNullOrZero(Integer number) {
     }
 
     private static Site modifySite(Site site, String trackingId) {
-        return site != null ? site.toBuilder().id(trackingId).build() : null;
+        return site != null ? site.toBuilder().id(Objects.toString(trackingId, "")).build() : null;
```

Comment:
Minor. Not the issue but just to inform, in this case we may use `ObjectUtils.defaultIfNull(..)` since `trackingId` is a string value.